src/components/KTable/KTable.vue

```diff
@@ -977,11 +1086,24 @@ export const defaultSorter = (key: string, previousKey: string, sortOrder: strin
 
   th,
   td {
+    border-right: var(--kui-border-width-20, $kui-border-width-20) solid $kui-color-border-transparent;
     padding: var(--kui-space-50, $kui-space-50) var(--kui-space-60, $kui-space-60);
     vertical-align: middle;
     white-space: nowrap;
   }
 
+  th.resize-hover {
+    border-right: var(--kui-border-width-20, $kui-border-width-20) solid var(--kui-color-border-decorative-purple, $kui-color-border-decorative-purple);
+  }
+
+  td.resize-hover {
+    border-right: var(--kui-border-width-20, $kui-border-width-20) solid rgba(175, 183, 197, .2); /** $kui-color-border as rgb */
```

Comment:
I tried the `neutral` shades but `weakest` was either too subtle or too strong. I'm happy to use whatever token you think looks best. I just used opacity because it was quickest to get the shade I was trying to have.